scenarios/kubernetes_e2e.py

```diff
@@ -656,6 +678,8 @@ def create_parser():
     parser.add_argument(
         '--kubeadm', choices=['ci', 'periodic', 'pull', 'stable'])
     parser.add_argument(
+        '--kubelet', default=None, help='kubelet version to use if it differs from kubeadm')
```

Comment:
Please do not add flags to the scenario.

Will you move all this new kubernetes_e2e.py directly into kubetest?

Reply:
Done (in new PR)